skl2onnx/common/onnx_optimisation_identity.py

```diff
@@ -0,0 +1,116 @@
+"""
+Optimization of :epkg:`ONNX` graphs.
+Functions in *onnxconverter-common* do not support
```

Comment:
onnxconverter-common does support opset.version < 9 with a limited optimization capability. But I think identity clean up should be included.

Reply:
It failed when I tried to use it.

Reply:
Can you the pre-optimized ONNX model?  we can take a look and fix it if any.

Reply:
Ok. I'll switch to onnxconverter_common optimizer and go through all unit tests.